src/LuaTransformer.ts

```diff
@@ -2671,10 +2671,26 @@ export class LuaTransformer {
         return this.transformBinaryOperation(typeCall, comparedExpression, operator, tsOriginal);
     }
 
+    protected transformComparisonExpression(expression: ts.BinaryExpression): ExpressionVisitResult {
```

Comment:
Did you just rename this? I could have sworn it was `transformTypeOfComparisonExpression` before (which I prefer)

Reply:
No - `transformTypeOfLiteralComparison` is above this function (I didn't change the name from before)